src/base-element.js

```diff
@@ -153,6 +153,20 @@ export class BaseElement {
     return false;
   }
 
+  /**
+   * Subclasses can override this method to indicate that an element can load
+   * network resources.
+   *
+   * Such elements can have their `ensureLoaded` method called.
+   *
+   * @param {!AmpElement} unusedElement
+   * @return {boolean}
+   * @nocollapse
+   */
+  static load(unusedElement) {
```

Comment:
Nit: `loadsWithReadyState` or `usesReadyState`. `loads` is super generic.

Reply:
I renamed it to `usesLoading`. Works?